src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -186,14 +185,22 @@ void buildHearingPartiesDetailsOtherParties() {
     }
 
     @DisplayName("When a valid hearing wrapper with joint party given buildHearingPartiesDetails returns the correct Hearing Parties Details")
-    @ParameterizedTest
-    @EnumSource(value = YesNo.class)
+    @Test
     @NullSource
-    void buildHearingPartiesDetailsJointParty(YesNo jointParty) {
-        // TODO SSCS-10378 - Finish Test
+    void buildHearingPartiesDetailsJointParty() {
```

Comment:
Can you add assertions for joint party to this test?

Reply:
okay , added

Reply:
Isn't this test made redundant by the following one?